### PR TITLE
Updating KubeVirt Tekton user guide page with current links

### PR DESCRIPTION
**Pull request by xphyr**
**What this PR does / why we need it**:
Windows Tekton Pipeline Example links in the current KubeVirt users guide do not point to working pages. This PR updates the links to point to the move source in the https://github.com/kubevirt/kubevirt-tekton-tasks/ repository.

This fixes Issue #790 

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by dhiller — COMMENTED** [review]
/approve

@ksimon1 mind taking a 2nd look here?

File: docs/virtual_machines/tekton_tasks.md [hunk 1]
```diff
@@ -97,12 +97,11 @@ KubeVirt-specific Tekton tasks, which are focused on:
...
 SSP is creating multiple example pipelines, e.g.:
 
-- [Windows BIOS installer](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-bios-installer-pipeline.yaml) - Pipeline will prepare a template and Windows datavolume vith virtio drivers installed. User has to provide a link to working Windows 10 iso file. Pipeline is suitable 
-for Windows versions, which uses BIOS. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-bios-installer)
+- [Windows BIOS installer](https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/templates-pipelines/windows-bios-installer/manifests/windows-bios-installer.yaml) - Pipeline will prepare a template and Windows datavolume vith virtio drivers installed. User has to provide a link to working Windows 10 iso file. Pipeline is suitable for Windows versions, which uses BIOS. More information about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/templates-pipelines/windows-bios-installer/README.md)
```
Comment (ksimon1) [inline docs/virtual_machines/tekton_tasks.md:100]: Correct link should be https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/release/pipelines/windows-bios-installer/windows-bios-installer.yaml.
The template-pipelines folder just contains templates. The result manifest are in release/pipelines.
Other links should point to release/pipelines too.

**Review by ksimon1 — CHANGES_REQUESTED** [review]
Thank you for you PR :+1: 
The links  in this PR are not correct and needs to be updated.

**Pull request merged**